lib/locales/en.yml

```diff
@@ -23,6 +23,10 @@ en:
   eoc_key_concepts: Key Concepts
   eoc_key_equations: Key Equations
   eoc_suggested_reading: Suggestions for Further Study
+  eoc: {
+    review-questions: Review Questions,
+    critical-thinking: Critical Thinking Questions
+  }
```

Comment:
Why are these added if they aren't used in this PR?

Reply:
@jpslav In my recipe, I'm using `MoveExercisesToEOC.v1` and passing these values to it. I included it in this PR because both of these sections have solutions in my answer key.